rotkehlchen/rotkehlchen.py

```diff
@@ -696,10 +696,10 @@ def remove_single_blockchain_accounts(
             blockchain=blockchain,
             accounts=accounts,
         )
-        eth_addresses: list[ChecksumEvmAddress] = cast(list[ChecksumEvmAddress], accounts) if blockchain == SupportedBlockchain.ETHEREUM else []  # noqa: E501
         with contextlib.ExitStack() as stack:
-            if blockchain == SupportedBlockchain.ETHEREUM:
-                ethereum = self.chains_aggregator.get_chain_manager(SupportedBlockchain.ETHEREUM)
+            if blockchain in {SupportedBlockchain.ETHEREUM, SupportedBlockchain.OPTIMISM}:
+                ethereum = self.chains_aggregator.get_chain_manager(blockchain)  # type: ignore[arg-type]  # noqa: E501
+                eth_addresses: list[ChecksumEvmAddress] = cast(list[ChecksumEvmAddress], accounts)
```

Comment:
Generalize the names

```suggestion
                evm_manager = self.chains_aggregator.get_chain_manager(blockchain)  # type: ignore[arg-type]  # noqa: E501
                evm_addresses: list[ChecksumEvmAddress] = cast(list[ChecksumEvmAddress], accounts)
```